src/resources/postcss/tickets-admin-settings.pcss

```diff
@@ -14,18 +14,16 @@
 	p.event-tickets__admin-banner-help-text {
 		max-width: 80%;
 	}
+}
 
+.event-tickets__admin-banner-help-links-wrapper {
+	display: flex;
 
-	.event-tickets__admin-banner-help-links {
-		display: flex;
-
-		div {
-			min-width: 50%;
-		}
-	}
-
-	.event-tickets__admin-banner-et-links h3,
-	.event-tickets__admin-banner-etp-links h3 {
-		font-size: 14px;
+	div {
+		min-width: 50%;
 	}
 }
+
+.event-tickets__admin-banner-links h3 {
```

Comment:
Should this be within the `.event-tickets__admin-banner` for structure or does that not matter?